docs/OV_Runtime_UG/hetero_execution.md

```diff
@@ -31,6 +31,8 @@ Affinities can be set in one of two ways, used separately or in combination: wit
 #### The Manual Option
 It assumes setting affinities explicitly for all operations in the model using `ov::Node::get_rt_info` with the `"affinity"` key. 
 
+If you set affinity for a specific operation, it may lead to a decrease in the accuracy of the model. If you encounter a decrease in accuracy, please try to set the related operations or related subgraphs of this operation to the same affinity, such as the constant operation that will be folded into this operation.
```

Comment:
I suppose that we need to write: make sure that if you assign specific layers to a specific devices, that the device actually supports the operation.

Current statement looks very strange: from user side I can think that it's allowed for OpenVINO to work incorrectly. But this is not true

Reply:
Maybe: "If you set affinity for a specific operation, it may lead to a decrease in the accuracy of the model. To avoid that, make sure that you...."